### ISSUE TITLE
QueryParams are parsed as object, not mutiple params

### ISSUE THREAD

**Issue by atomicus** [epiphone/routing-controllers-openapi#39]
This looks similiar to #38, but I'm not doing any kind of merging here.

Having simple:
```
class PageableQuery {
    @IsNumber()
    @Max(50)
    @IsPositive()
    per_page: number = 50;

    @IsNumber()
    @IsPositive()
    page: number = 1;

    username: string;
}

#....
    @Get('/')
    public async getAll(@QueryParams() query: PageableQuery) {
}
```

Generates swagger with one query param of object type, instead of mutiple query params like this:
<img width="373" alt="Zrzut ekranu 2020-02-6 o 21 51 01" src="https://user-images.githubusercontent.com/188840/73977530-d9d09100-492a-11ea-9482-5015da874e5d.png">

I'm doing something wrong?

**Comment by epiphone** [comment]
No, that's by design. I'd be happy to review a PR to change the behaviour though: we'd have to iterate through the `queriesMeta` properties [here](https://github.com/epiphone/routing-controllers-openapi/blob/master/src/generateSpec.ts#L154) and do a separate `queries.push({ in: 'query', ... })` for each property, or something similar.

**Comment by atomicus** [comment]
Sure, i'll give a try

**Comment by epiphone** [comment]
Released in v2.0.1
